pkgs/by-name/ha/hap-py/package.nix

```
@@ -0,0 +1,63 @@
+{ pkgs, stdenv, fetchFromGitHub, boost, zlib, lib, makeWrapper }:
+
+let
+  # Bcftools needs perl
+  runtime = with pkgs; [ bcftools htslib my-python perl samtools ];
+  my-python-packages = p: with p; [
+    bx-python
+    pysam
+    cython
+    pandas
+    psutil
+    nose
+    scipy
+  ];
+  my-python = pkgs.python3.withPackages my-python-packages;
+in
+stdenv.mkDerivation rec {
+  pname = "hap.py";
+  version  = "0.3.15";
+  src = fetchFromGitHub {
+    owner = "Illumina";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-K8XXhioMGMHw56MKvp0Eo8S6R36JczBzGRaBz035zRQ=";
+  };
+  # For illumina script
+  BOOST_ROOT = "${boost.out}";
+  ZLIBSTATIC = "${zlib.static}";
+  # For cmake : boost lib and includedir are in differernt location
+  BOOST_LIBRARYDIR = "${boost.out}/lib";
+  BOOST_INCLUDEDIR = "${boost.dev}/include";
+  patches = [
+    ./hap-py.patch
+    ./cmake.patch
+  ];
```

```suggestion

  patches = [
    ./hap-py.patch
    ./cmake.patch
  ];

```
Why do we need those patches?